src/component/list.js

```diff
@@ -117,6 +124,76 @@ ListHeader.propTypes = {
   style: View.propTypes.style,
 };
 
+//
+// Setting Content
```

Comment:
Maybe we should move all the `Setting` components into their own component file?

Reply:
> Maybe we should move all the `Setting` components into their own component file?

Good call. I moved the settings components into their own file in https://github.com/lightninglabs/lightning-app/pull/839/commits/f8c8ca456fed1f3071c8b0c0fe506c82cddd364a.

> Wow, that got rid of a ton of code! Very nice. Minor suggestion but feel free to merge.

Cool 🎉 Merging :)